linked_list/LinkedList.c

```diff
@@ -0,0 +1,176 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct Node {
+	int d;		//	integer type data  
+	struct Node* next;		//	pointer to same structure
+}*start;	//	first node of the list
```

Comment:
Try to be consistent with the comments align them equally or create a multi line comment, showing properties of each attribute.